logging/source/RialtoLogging.cpp

```diff
@@ -110,7 +110,8 @@ static void consoleLogHandler(RIALTO_DEBUG_LEVEL level, const char *file, int li
     iov[3].iov_len = messageLen;
     iov[4].iov_base = const_cast<void *>(reinterpret_cast<const void *>("\n"));
     iov[4].iov_len = 1;
-    writev(STDERR_FILENO, iov, 5);
+    // TODO(unknown): consider using standard write(2) and handle EINTR properly.
```

Comment:
We don't like to leave TODOs in the code, because they have tendency to stay there forever. 
Please ask @skypetts to add it to the backlog.

Reply:
I've raised https://jira.rdkcentral.com/jira/browse/RIALTO-38 for this